### ISSUE TITLE
Feature request: Exclude files

### ISSUE THREAD

**Issue by fluzz** [cobbzilla/s3s3mirror#49]
Are they any plans to add inclusion/exlusion of files 

so that we can only sync files that either match or dont match a supplied regex?

**Comment by cobbzilla** [comment]
Yes. Wouldn't be too hard to add. I would happily accept a pull request that implements this feature.
